server/osint/tools/lib/tool_maigret.py

```diff
@@ -56,10 +58,21 @@ def report_maigret(run):  # status is COMPLETED
         "state": run.status,
         "results": result_list
     }
+
+    # node
+    s_user = SurfaceUser.get_node({'username': run.input_value, 'case_id': case_id})
+    if not s_user:
+        s_user = SurfaceUser.created_node({'username': run.input_value, 'case_id': case_id})
```

Comment:
Is it **create_node()** not created_node()?

Reply:
oh shoot. but it worked... how...?